in_toto/verifylib.py

```diff
@@ -232,15 +233,15 @@ def verify_all_steps_signatures(layout, links_dict):
   """
   for step in layout.steps:
     # Find the according link for this step
-    link = links_dict[step.name]
-
+    key_link_dict = links_dict[step.name]
```

Comment:
For now, I have used the following names for the dicts :-
1. "chain_link_dict" for the whole dict with step-names as keys and values that are
2. "key_link_dict" for the inner dict with functionary keyids as keys and Link objects as values
3. "reduced_chain_link_dict" for the condensed dict with step names as keys and Link ojects as values

We can decide on this! It won't be a big deal to change these names again :-)